assets/webconfig/js/content_leds.js

```diff
@@ -1659,18 +1663,21 @@ $(document).ready(function () {
       optArr[4].push(ledDevices[idx]);
     else if ($.inArray(ledDevices[idx], devHID) != -1)
       optArr[4].push(ledDevices[idx]);
-    else
+    else if (ledDevices[idx].endsWith("_ftdi"))
```

Comment:
Just a suggestion to have the titles in the dropdown without suffix, as the grouping already says ftdi

```
    else if (ledDevices[idx].endsWith("_ftdi")) {
      var title = ledDevices[idx].replace('_ftdi','');
      optArr[5].push(ledDevices[idx] + ":" + title);
      }
```